scrapy/utils/test.py

```diff
@@ -87,7 +87,7 @@ def get_crawler(
     # Set by default settings that prevent deprecation warnings.
     settings: Dict[str, Any] = {}
     if prevent_warnings:
-        settings["REQUEST_FINGERPRINTER_IMPLEMENTATION"] = "2.7"
+        pass
```

Comment:
Just remove these lines? :)

Reply:
Seems that work as well, local pre-commit pass, cc: @Gallaecio

Reply:
(if some linters complain and these lines are needed, we should add some comments to them to explain why they are needed)